nuxt/pages/reciters/_id/albums/_albumId/index.vue

```diff
@@ -0,0 +1,354 @@
+      ],
+    });
+
+    this.tracks = response.data;
+
+    this.setBackgroundFromImage();
+  },
+
+  data(): Data {
+    return {
+      album: null,
+      background: 'rgb(47,47,47)',
+      textColor: '#fff',
+      track: null,
+      tracks: null,
+      addedToQueueSnackbar: false,
+    };
+  },
+
+  computed: {
+    ...mapGetters('auth', ['isModerator']),
+    reciter(): Reciter|null|undefined {
+      return this.album && this.album.reciter;
+    },
+    image(): string {
+      if (this.album) {
+        return this.album.artwork || '/defaults/default-album-image.png';
+      }
+      return '/defaults/default-album-image.png';
+    },
+    playable(): Array<any> {
+      if (!this.tracks) {
+        return [];
+      }
+
+      return this.tracks.filter((track) => this.hasAudioFile(track));
+    },
+    hasPlayableTracks(): boolean {
+      return this.playable.length > 0;
+    },
+    heroArtworkSize(): number {
+      if (this.$vuetify.breakpoint.xsOnly) {
+        return 96;
+      }
+      if (this.$vuetify.breakpoint.smOnly) {
+        return 128;
+      }
+      return 192;
+    },
+    isDark(): boolean {
+      return this.$vuetify.theme.dark;
+    },
+  },
+
+  watch: {
+    $route: 'onRouteChanged',
+  },
+
+  methods: {
+    setBackgroundFromImage() {
+      if (!this.album) {
+        return;
+      }
+      Vibrant.from(this.image)
+        .getPalette()
+        .then((palette) => {
+          const swatch = palette.DarkMuted;
+          if (!swatch) {
+            return;
+          }
+          this.background = swatch.getHex();
+          this.textColor = swatch.getBodyTextColor();
+        });
+    },
+    isSameAlbum({ reciter, album }): boolean {
+      if (!this.track || this.track.reciter === undefined || this.track.album === undefined) {
+        return false;
+      }
+      return (
+        this.track.reciter.slug === reciter &&
+        this.track.album.year === album
+      );
+    },
+    getTrackLink(track) {
+      if (!this.reciter || !this.album) {
+        return;
+      }
+      return `/reciters/${this.reciter.slug}/albums/${this.album.year}/tracks/${track.slug}`;
+    },
+    hasAudioFile(track): boolean {
+      return track.related ? track.related.audio : false;
+    },
+    hasLyrics(track): boolean {
+      return track.related ? track.related.lyrics : false;
+    },
+    playAlbum() {
+      this.$store.commit('player/PLAY_ALBUM', { tracks: this.playable, start: this.playable[0] });
+    },
+    playTrack(track) {
+      this.$store.commit('player/PLAY_ALBUM', { tracks: this.playable, start: track });
+    },
+    addToQueue() {
+      this.$store.commit('player/ADD_ALBUM_TO_QUEUE', { tracks: this.playable });
+      this.addedToQueueSnackbar = true;
+    },
+    onRouteChanged() {
+      this.$vuetify.goTo(0);
+      this.$fetch();
+    },
+  },
+
+  head(): MetaInfo {
+    const title = `${this.album?.reciter?.name} - ${this.album?.year} Album - ${this.album?.title}` ?? 'Album';
```

Comment:
This line of code doesn't make sense. The null coalescing operator `??` won't ever trigger, because the string will never be `null`.

What you probably want here is something like:

```ts
let title = `${this.$route.params.albumId} Album`;

if (this.album) {
    title = `${this.album?.reciter?.name} - ${this.album?.year} Album - ${this.album?.title}`
}

return { title };